gke/standard/zonal/secondary-boot-disk/main.tf

```diff
@@ -45,3 +45,34 @@ resource "google_container_node_pool" "default" {
   }
 }
 # [END gke_standard_zonal_secondary_boot_disk]
+
+# [START gke_standard_zonal_secondary_boot_disk_data]
+resource "google_container_cluster" "default" {
+  name               = "default"
+  location           = "us-central1-a"
+  initial_node_count = 1
+  # Set `min_master_version` because secondary_boot_disks require GKE 1.28.3-gke.106700 or later
+  min_master_version = "1.28"
+  # Set `deletion_protection` to `true` will ensure that one cannot
+  # accidentally delete this instance by use of Terraform.
+  deletion_protection = false
+}
+
+resource "google_container_node_pool" "default" {
```

Comment:
If there are two node pools, rather than `default` you will need to use a dedicated name for each, and I recommend matching to the tag `[START gke_standard_zonal_secondary_boot_disk_data]` `[End gke_standard_zonal_secondary_boot_disk_data]`

Reply:
Changed node pool names to be different